neptune/alpha/internal/backends/neptune_backend_mock.py

```diff
@@ -114,6 +116,9 @@ def create_experiment(self,
             self._experiments[new_experiment_uuid].set(["source_code", "git"], git_ref)
         return ApiExperiment(new_experiment_uuid, short_id, 'workspace', 'sandbox', False)
 
+    def create_checkpoint(self, notebook_id: uuid.UUID, jupyter_path: str) -> Optional[uuid.UUID]:
+        return None
```

Comment:
How about NotImplementedError?

Reply:
This is a mock/stub/offline version and notebooks are "optional" so they should not throw on non-hosted backend